src/ai/backend/manager/models/vfolder.py

```diff
@@ -204,6 +204,18 @@ class VFolderOperationStatus(enum.StrEnum):
     DELETE_COMPLETE = "delete-complete"  # vfolder is deleted permanently, only DB row remains
     DELETE_ERROR = "delete-error"
 
+    def is_deletable(self, force: bool = False) -> bool:
+        if force:
+            return self in {
+                VFolderOperationStatus.READY,
+                VFolderOperationStatus.DELETE_PENDING,
+            }
```

Comment:
Even with `force` option, does it mean that only the `ready` and `delete_pending` states are deletable?

Reply:
yes, `force` option allows deleteing vfolders in `ready` state.

Reply:
Updated this to allow `DELETE_ERROR` and `DELETE_ONGOING` status